src/components/Addons/Collapse/Collapse.tsx

```diff
@@ -50,7 +50,7 @@ class Collapse extends React.PureComponent<CollapseProps & React.HTMLAttributes<
 
   public render() {
     const { children, className, defaultClosed, title, TitleComponent = 'div', variant, ...attrs } = this.props;
-    const classes = classNames({
+    const classes = classNames.default({
```

Comment:
Why was this `.default` added? Is it a TS thing? I'm not seeing it in the `classNames` docs

Reply:
I think it's a TS thing. This happened following the update of the `react-scripts` and `typescript` dependencies. It's something to do with the way the exports are set up on `classNames` typings. I tried several things and this is the only thing I could figure out that made the compiler happy. JS/TS imports and exports are baffling to me. I'm happy to take a different approach if you can recommend one.

Reply:
@jhoerr, we fixed this issue earlier by importing classnames as follows: `import classNames from 'classnames';`. Just an FYI

Reply:
Fixed with @nublikasa's advice.